pyproject.toml

```diff
@@ -1,11 +1,30 @@
 [build-system]
 requires = [
   # sync with setup.py until we discard non-pep-517/518
-  "setuptools>=45.0",
-  "setuptools-scm[toml]>=6.2.3",
+  "setuptools>=65.0",
+  "setuptools-scm[toml]>=8.0",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pluggy"
+license = {text = "MIT"}
+authors = [{name = "Holger Krekel", email = "holger@merlinux.eu"}]
```

Comment:
While at it should we update this? Not sure to what exactly though...

Reply:
I'm up for being on the hook

Reply:
I'm up for being on the hook

Reply:
I was thinking more if it is appropriate to add `pytest core team <core@pytest.org>` or something like that.